bylaws.md

```diff
@@ -613,6 +619,18 @@ has a significant consulting relationship with an entity; or that
 person owns at least 1% of the equity or debt, or derivatives 
 thereof, of an entity.
 
+**Section 5.15. Limits on Co-affiliation of Board Members.** No more
+than one quarter (1/4) of the members of the Board of Directors may
+share a common affiliation as defined in Section 5.14. If the number
+of co-affiliated directors goes above the limit due to a change in
+employment or a corporate acquisition, then the longest-serving member(s)
+of the Board of Directors sharing that affiliation must resign before
```

Comment:
Ambiguity here permits that to be a flexible exercise of who resigns and how I presume. So if say a person's parent company is acquired, pushing that person to be more than 1/4 of the BoD membership, this ambiguity then leads me to believe that:
- If a person's parent company changes in July of a year
- They don't have to resign until the *latest* being the following May (since June historically is when elections are held)
Is that correct?

Reply:
Yes, and the motivations you inferred are correct also. Even if a single affiliation group goes above 25%, it is still a minority on the board, and it is better to force a change at the next election than force an immediate withdrawal of a director.

But thinking about it, if the board agrees that forcing immediate withdrawal is not an issue, we could actually just delete the sentence about resigning, and just keep the last sentence about co-affiliates over the limit not being eligible to be seated or appointed.